api/internal/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// packages under `api/internal` are for internal use only and may be subject to breaking changes.
```

Comment:
I don't know if you even need this comment. I suggested `internal/` because the compiler enforces this.

https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface

Reply:
Woah, TIL.